newton/_src/solvers/style3d/collision/collision.py

```
@@ -0,0 +1,278 @@
+        """
+        Refit (update) triangle and edge BVHs based on new positions without changing topology.
+
+        Args:
+            pos: Array of vertex positions.
+            tri_indices: Array of triangle vertex indices.
+            edge_indices: Array of edge vertex indices.
+        """
+        self.tri_bvh.refit(pos, self.model.tri_indices, self.radius)
+        self.edge_bvh.refit(pos, self.model.edge_indices, self.radius)
+
+    def frame_begin(self, particle_q: wp.array(dtype=wp.vec3), vel: wp.array(dtype=wp.vec3), dt: float):
```

Ruff (0.13.1)

Unused method argument: `vel`

(ARG002)